common/src/main/java/com/wynntils/wynn/objects/ClassInfo.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.wynn.objects;
+
+import net.minecraft.world.item.ItemStack;
+
+public record ClassInfo(
```

Comment:
I think this overlaps quite a bit with `CharacterInfo` in `CharacterManager`. Can we unify them?

Reply:
Also, I think we should stick to the name "Character" for selecting the character to play, not "class" (and keep that strictly as a name for the kind of "style" the character is.) Even Wynncraft themselves have renamed the dialog to "Select a Character", not "Select a Class". (Still the command is `/class` but I assume old habits die hard.)

Reply:
Does it? They are used for different things, parsed differently. We parse different things for both of them because the other info is not needed.

Reply:
I am fine with renaming though.. But then, it kinda conflicts.. And I am not sure I like them merged

Reply:
The one thing that this class contains which I'm not 100% happy about putting in CharacterInfo is the slot id and itemstack. I understand why you need it here, but it should not really be exposed later on with the CharacterInfo.

Reply:
If you are okay with it, I can accept the code as it is now, and then see if I can merge them in a good way, so that the slot is exposed only where it should be.

Reply:
You could technically get the itemstack from the container with the slot id.. Not sure how to remove both.

Reply:
And sure, I don't really have an idea for refactoring this anyways

Reply:
Let's go with this for now, and I'll put it on my mental todo-list of things I want to fix later on :)